src/main/webapp/WEB-INF/tags/student/comments/feedbackSessionPanel.tag

```diff
@@ -6,8 +6,11 @@
 <div class="panel panel-primary">
     <div class="panel-heading">
         <strong>Comments in session: ${feedbackSessionRow.feedbackSessionName}</strong>
+        <div class='display-icon pull-right'>
```

Comment:
On the Student comments page, there are 2 types of comments panels, the general student comments panel and the response comments panel. This tag corresponds to the response comments panel, which isn't the panel that we want to collapse (in this issue at least). Changing the commentsPanel tag is sufficient as it is the common tag that is used by the 4 pages to show the student comments panel.